Assets/Plugins/Android/Core/AndroidConfig.cs

```diff
@@ -25,8 +25,21 @@ namespace PlayEveryWare.EpicOnlineServices
     // Flags specifically for Android
     public class AndroidConfig : PlatformConfig
     {
+        [ConfigField("Google Login Client ID",
+            ConfigFieldType.Text,
+            "Get your project's Google Login Client ID from the " +
+            "Google Cloud dashboard",
+            -1,
+            "https://console.cloud.google.com/apis/dashboard")]
         public string GoogleLoginClientID;
+
+        [ConfigField("Google Login Nonce",
+            ConfigFieldType.Text,
+            "Use a nonce to improve sign-ing security on Android.",
```

Comment:
Shall tooltips end in punctuation, or shall they end without it?

Reply:
They should end in punctuation, they use standard grammar like everything else IMO.

Reply:
It's the smallest of nits, but there's some inconsistent application of punctuation in these tooltips. Consider adjusting them. Resolve this conversation at your leisure either way, it's definitely just a nit.